docs/storybook/src/components/MessageCenterField.stories.tsx

```diff
@@ -34,4 +41,153 @@ const meta = {
 export default meta;
 type Story = StoryObj<typeof meta>;
 
-export const Basic: Story = {};
+export const Empty: Story = {};
+
+//Info UseCase
+const NotificationDecorator: Decorator = (Story) => {
+  React.useEffect(() => {
+    MessageManager.clearMessages();
+    ["1", "2", "3", "4"].forEach((num) => {
+      MessageManager.addToMessageCenter(
+        new NotifyMessageDetails(OutputMessagePriority.Info, `Message ${num}`)
+      );
+    });
+  });
+
+  return (
+    <div>
+      <Story />
+    </div>
+  );
+};
+
+export const Notification: Story = {
+  decorators: NotificationDecorator,
+};
+
+//Error Use Case
+
+const ErrorDecorator: Decorator = (Story) => {
+  React.useEffect(() => {
+    MessageManager.clearMessages();
+    ["1", "2", "3", "4"].forEach((num) => {
+      MessageManager.addToMessageCenter(
+        new NotifyMessageDetails(OutputMessagePriority.Error, `Error ${num}`)
+      );
+    });
+  });
+
+  return (
+    <div>
+      <Story />
+    </div>
+  );
+};
+
+export const Error: Story = {
+  decorators: ErrorDecorator,
+};
+
+//Detailed Message
+
+const DetailedDecorator: Decorator = (Story) => {
+  React.useEffect(() => {
+    MessageManager.clearMessages();
+    [1, 2, 3, 4].forEach((num) => {
```

Comment:
Why only last message color is green & others are rendered w/ black color?
![Screenshot 2024-04-04 at 10 08 05](https://github.com/iTwin/appui/assets/10091419/487dcde5-6fd8-49f5-a77a-e367dc4ec4c4)